client/pages/games.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+
+export default class Games extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      games: [],
+      lists: []
+    };
+    this.newGame = this.newGame.bind(this);
+    this.getGame = this.getGame.bind(this);
+    this.getWords = this.getWords.bind(this);
+  }
+
+  componentDidMount() {
+    fetch('api/games')
```

Comment:
All of your `fetch` request paths should start with a `/` so this one should be `/api/games`. While this works okay in development, without the slash it is likely to break in production.

Reply:
got it, on it 👍